### PR TITLE
LAUN-462: OIDC SDK Publishing Platform Migration

### PR DESCRIPTION
**Pull request by trishHUYNH**
- Remove Bintray publishing scripts
- Update GHA workflow to publish to Maven Central
- Add publishing gradle scripts
- Update build gradle scripts

### CODE REVIEW

File: scripts/publish-module.gradle [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                    url = 'https://github.com/onelogin/onelogin-oidc-android'
+                    licenses {
+                        license {
+                            name = 'MIT'
+                            url = 'https://github.com/onelogin/onelogin-oidc-android/blob/master/LICENSE'
+                        }
+                    }
+                    developers {
+                        developer {
+                            id = 'TrishIsAtOneLogin'
```
Comment (codepete) [inline scripts/publish-module.gradle:57]: We might want to move this a generic email address that people can reach out to - support maybe? @ronc or @jcbarnett

Comment (trishHUYNH) [reply scripts/publish-module.gradle:57]: Fine by me that someone else gets spammed. :)

**Review by CarleenSinton — APPROVED** [review]

**Review by codepete — APPROVED** [review]
👍🏼

**Review by CarleenSinton — APPROVED** [review]
Looks good!

**Pull request merged**